modules/dnn/test/test_layers.cpp

```diff
@@ -141,7 +141,13 @@ TEST_P(Test_Caffe_layers, Convolution)
 
 TEST_P(Test_Caffe_layers, DeConvolution)
 {
-    testLayerUsingCaffeModels("layer_deconvolution", true, false);
+    double l1 = 0.0, lInf = 0.0;
+    if(target == DNN_TARGET_CUDA_FP16)
+    {
+        l1 = 4;
+        lInf = 60;
```

Comment:
Please also specify reference values range.

Reply:
Is it ok if I leave deconvolution tests failing? There is a problem which has been there ever since the original CUDA backend had been merged.

Reply:
Why not skip them?

Reply:
I thought broken stuff should be failing tests instead of being hidden. I'll skip then.

Reply:
So, we just trying to follow green tests strategy. To ignore skip tag there is a flags (maybe `--test_tag_enable` if I'm correct).